app/build.gradle

```diff
@@ -72,14 +73,14 @@ dependencies {
   }
 
   androidTestImplementation "com.android.support:support-annotations:$supportLibraryVersion"
-  androidTestImplementation 'com.android.support.test:runner:1.0.1'
-  androidTestImplementation 'com.android.support.test:rules:1.0.1'
+  androidTestImplementation 'com.android.support.test:runner:1.0.2'
+  androidTestImplementation 'com.android.support.test:rules:1.0.2'
 
   // Tab indicator
   implementation "com.pacioianu.david:ink-page-indicator:$inkPageIndicatorVersion"
 
   // Guava
-  implementation group: 'com.google.guava', name: 'guava', version: '21.0'
```

Comment:
Revert the changes made in this gradle file the build will pass. Use 2 space tabs everywhere instead of 4.